setup.py

```diff
@@ -50,7 +50,7 @@
                       'six==1.11.0', 'psutil==5.4.3', 'intervaltree==2.1.0',
                       'msgpack-python==0.4.6', 'indy-crypto==0.4.3',
                       'python-rocksdb==0.6.9', 'python-dateutil==2.6.1',
-                      'pympler==0.5'],
+                      'pympler==0.5', 'bitarray==0.8.1'],
```

Comment:
Need to have a deb package as well?

Reply:
No, there is python3-bitarray=0.8.1 package in the xenial repo